jobs/cjd_sample/src/main/java/com/google/samples/BatchOperationSample.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright 2018 Google Inc.
```

Comment:
New files as of 2018 should use "Google LLC" instead of "Google Inc." in their copyright headers. Ditto for the other files.

Reply:
Done.